client/starwhale/base/scheduler/__init__.py

```diff
@@ -42,7 +42,7 @@ def run(
             return self._schedule_all()
 
         if task_index is None or task_index < 0:
-            return [self._schedule_one_step(step_name=step_name, task_num=task_num)]
+            return [self._schedule_one_step(step_name=step_name)]
```

Comment:
Tasknum in schedule one step function was used to override the stepSpec's 'task_num', Is it should not be deleted?